### DISCUSSION TITLE
cc1plus: fatal error: knn.cxx: No such file or directory

### DISCUSSION THREAD

**Discussion by nicetomeet-you** [QingyongHu/RandLA-Net#275] [Q&A]
recently,im running compile_op.sh like this:
`
cd utils/nearest_neighbors
python setup.py install --home="."
cd ../../

cd utils/cpp_wrappers
sh compile_wrappers.sh
cd ../../../
`

AND i get this:
`
running install
running build
running build_ext
building 'nearest_neighbors' extension
/usr/bin/gcc -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I./ -I/root/miniconda3/lib/python3.8/site-packages/numpy/core/include -I/root/miniconda3/include/python3.8 -c knn.cxx -o build/temp.linux-x86_64-3.8/knn.o -std=c++11 -fopenmp
cc1plus: warning: command-line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
cc1plus: fatal error: knn.cxx: No such file or directory
compilation terminated.
error: command '/usr/bin/gcc' failed with exit status 1
`
It told me knn.cxx is miss ,so where is it? or how can i solve this issue?

**Comment by TangSui122** [comment]
utils/nearest_neighbors/knn.pyx:
# distutils: sources = knn.cxx
replace knn.cxx by knn_.cxx

**Reply by 1799749390** [reply]
牛逼

**Reply by ypqlvkxx** [reply]
牛逼

**Reply by Chongyu-hub** [reply]
bro you are the real hero

**Reply by Horace714** [reply]
牛逼

**Reply by cj916** [reply]
牛逼，直接改不用管#的
